syntaxhighlighter.php

```diff
@@ -659,16 +677,22 @@ function shortcode_hack( $content, $callback, $ignore_html = true ) {
 			return $content;
 		}
 
-		// Backup current registered shortcodes and clear them all out
-		$orig_shortcode_tags = $shortcode_tags;
+		// Backup current registered shortcodes and clear them all out (we do not backup our own, because we will add and parse them below)
+		$orig_shortcode_tags = array_diff_key( $shortcode_tags, array_flip( $this->shortcodes ) );
 		remove_all_shortcodes();
 
 		// Register all of this plugin's shortcodes
 		foreach ( $this->shortcodes as $shortcode ) {
 			add_shortcode( $shortcode, $callback );
 		}
 
-		$regex = '/' . get_shortcode_regex( $this->shortcodes ) . '/';
+		// Register all other shortcodes, ensuring their content remains unchanged using yet another hack.
+		foreach ( $orig_shortcode_tags as $shortcode_tagname => $shortcode ) {
+			add_shortcode( $shortcode_tagname, '__return_empty_string' );
+			add_filter( 'pre_do_shortcode_tag', array( $this, 'pre_do_shortcode_shortcode_hack_skip_others' ), 10, 4 );
```

Comment:
Because I made a small mistake there. We should be adding filters just once, thanks for catching that. Fixed this in https://github.com/Automattic/syntaxhighlighter/pull/261/commits/392bfae4dfc2860f061c87708845bd840b101dc7.